Automattic/wp-calypso
# Seller Experience: Add message for users who upgraded from payments block

#### Changes proposed in this Pull Request

* Added a Snackbar with a message for users who upgraded from Payments block.
Currently, it [checks for the notice parameter](https://github.com/Automattic/wp-calypso/issues/61421#issuecomment-1049231469) on the calypso level, passes it to the editor and the editor shows the modal.


#### Testing instructions



* Use a sandboxed site
* [Sync](https://github.com/Automattic/wp-calypso/tree/trunk/apps) calypso's `editing-toolkit` app to your sandbox
* Open a post editor `http://calypso.localhost:3000/post/<site>/<post-id>?notice=purchase-success`

<img width="1300" alt="Screen Shot 2022-06-06 at 12 42 24" src="https://user-images.githubusercontent.com/1234758/172154227-6efa0988-20d9-41a0-b5a5-63292416a032.png">



Closes #61421
